healthcheck/__init__.py

```diff
@@ -2,7 +2,7 @@
 __version__ = '0.0.7'
 __url__ = 'https://github.com/yola/healthcheck'
 
-
-from healthcheck import (HealthChecker, HealthCheck, ListHealthCheck,
+ROOT_URLCONF = ''
+from .healthcheck import (HealthChecker, HealthCheck, ListHealthCheck,
```

Comment:
E402 module level import not at top of file
F401 '.healthcheck.ListHealthCheck' imported but unused
F401 '.healthcheck.HealthCheck' imported but unused
F401 '.healthcheck.FilesExistHealthCheck' imported but unused
F401 '.healthcheck.HealthChecker' imported but unused
F401 '.healthcheck.DjangoDBsHealthCheck' imported but unused
F401 '.healthcheck.FilesDontExistHealthCheck' imported but unused
I101 Imported names are in the wrong order. Should be DjangoDBsHealthCheck, FilesDontExistHealthCheck, FilesExistHealthCheck, HealthCheck, HealthChecker, ListHealthCheck